README.md

```diff
@@ -66,81 +80,100 @@ eight_points_guzzle:
-            handler: 'GuzzleHttp\Handler\MockHandler'
-
             # guzzle client options (full description here: https://guzzle.readthedocs.org/en/latest/request-options.html)
-            # NOTE: "headers" option is not accepted here as it is provided as described above.
             options:
                 auth:
                     - acme     # login
                     - pa55w0rd # password
 
                 headers:
-                    Accept: "application/json"
+                    Accept: 'application/json'
 
                 # Find proper php const, for example CURLOPT_SSLVERSION, remove CURLOPT_ and transform to lower case.
                 # List of curl options: http://php.net/manual/en/function.curl-setopt.php
                 curl:
-                    sslversion: 1 # or !php/const:CURL_HTTP_VERSION_1_0 for symfony >= 3.2
+                    !php/const:CURL_HTTP_VERSION_1_0: 1
 
                 timeout: 30
 
             # plugin settings
             plugin: ~
 
-        api_crm:
-            base_url: "http://api.crm.tld"
+        crm:
+            base_url: 'http://api.crm.tld'
             options:            
                 headers:
-                    Accept: "application/json"
+                    Accept: 'application/json'
 
-        ...
+        # More clients here
 ```
 
-Open [Configuration Reference](src/Resources/doc/configuration-reference.md) page to see the complete list of allowed options.
+Please refer to the [Configuration Reference](src/Resources/doc/configuration-reference.md) for a complete list of all options.
 
-##### Install assets _(if it's not performed automatically)_:
-``` bash
-bin/console assets:install
+## Usage
+
+Guzzle clients configured through this bundle are available in the Symfony Dependency Injection container under the name
+`eight_points_guzzle.client.<name of client>`. So for example a client configured in the configuration with name `payment` is available
+as `eight_points_guzzle.client.payment`.
+
+Suppose you have the following controller that requires a Guzzle Client:
+
+```php
+<?php
+
+namespace App\Controller;
+
+use Guzzle\Client;
+
+class ExampleController
+{
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+}
 ```
 
-Using services in controller (eight_points_guzzle.client.**api_crm** represents the client name of the yaml config and is an instance of GuzzleHttp\Client):
+Using manual wiring this controller can be wired as follows:
 
-``` php
-/** @var \GuzzleHttp\Client $client */
-$client   = $this->get('eight_points_guzzle.client.api_crm');
-$response = $client->get('/users');
+```yaml
+services:
+    my.example.controller:
+        class: App\Controller\ExampleController
+        arguments: ['@eight_points_guzzle.client.payment']
 ```
 
+For projects that use autowiring[18], please refer to [our documentation on autowiring](src/Resources/doc/autowiring-clients.md).
```

Comment:
`autowiring` here is not as a link. Please change to `[autowiring][18]`.

Reply:
Oops! Nice catch. Fixed :slightly_smiling_face: